fearless/Modules/Pincode/LocalAuthentification/LocalAuthPresenter.swift

```diff
@@ -55,11 +55,17 @@ class LocalAuthPresenter: PinSetupPresenterProtocol {
 extension LocalAuthPresenter: LocalAuthInteractorOutputProtocol {
     func didEnterWrongPincode() {
         DispatchQueue.main.async { [weak self] in
+            self?.view?.didStopLoading()
```

Comment:
I didn't see 'didStopLoading' for any other case except wrong. please check and add if needed

Reply:
this is not needed, because if pin valid we join in the main screen